exercises/patterns/printHollowPyramid/printHollowPyramid.js

```diff
@@ -28,10 +28,27 @@ let helpers = require('../printHelpers');
  * @param {number} height - The height of the pyramid to print
  */
 function printHollowPyramid(height) {
-  for (let i = 0; i < height; i++) {
-    // This is your job. :)
-
+  let counter = 1;
+  let middleSpace = 1;
```

Comment:
This is probably easier if you separate out the first and last lines, to keep the conditionals out of the loop body.

Consider:

```javascript
let numSpaces = height - i;
let numMiddleSpaces = 2 * (i - 1) - 1;
```